src/InstagramClone.Api/Controllers/UsersController.cs

```diff
@@ -65,5 +62,27 @@ public IActionResult CreateUsers([FromBody] UserCreateDTO userDto)
             return CreatedAtAction(nameof(GetUser),new {id = userDto.Id },  userDto);
         } 
 
+        [HttpPost("follow")]
+        public IActionResult Follow([FromBody]FollowersDto followersDto)
+        {
+
+            // "helped me alot" https://www.youtube.com/watch?v=j1e6Z-7QNpk
+
+            var followed = followersDto.FollowedUserId;
+            var following = followersDto.FollowerId;
+            
+              var followersTofollow = new UserFollower()
+              {
+                  FollowerId = followersDto.FollowerId,
+                  FollowedUserId = followersDto.FollowedUserId,
+              };
+            this._context.UserFollowers.Add(followersTofollow);
+            this._context.SaveChanges();
```

Comment:
Can you check check your database and confirm that the createdDate column has the correct values after insertion.

Reply:
Ok